### PR TITLE
[MFTF] Added some ActionGroups to Analytics module

### LINKED ISSUE #29648: [Issue] [MFTF] Added some ActionGroups to Analytics module
This issue is automatically created based on existing pull request: magento/magento2#29634: [MFTF] Added some ActionGroups to Analytics module

---------
This PR adds `AdminConfigGeneralAnalyticsPageOpenActionGroup` and `AssertAdminAdvancedReportingPageUrlActionGroup`  ActionGroups to Analytics module

### PR DESCRIPTION
**Pull request by Usik2203**
This PR adds `AdminConfigGeneralAnalyticsPageOpenActionGroup` and `AssertAdminAdvancedReportingPageUrlActionGroup`  ActionGroups to Analytics module

### Resolved issues:
1. [x] resolves magento/magento2#29648: [MFTF] Added some ActionGroups to Analytics module

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

File: app/code/Magento/Analytics/Test/Mftf/ActionGroup/AdminConfigGeneralAnalyticsPageOpenActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminConfigGeneralAnalyticsPageOpenActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Analytics/Test/Mftf/ActionGroup/AdminConfigGeneralAnalyticsPageOpenActionGroup.xml]: I would recommend include a verb into the name that describes an action in the action group. It's a common practice and we will have more or less transparent naming convent in this case.

`AdminOpenConfigGeneralAnalytticsPageActionGroup`

**Comment by sidolov** [comment]
@magento create issue

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @Usik2203. Please take a look at builds that are failed and rename action group according to the recommendation provided by @rogyar.

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE ,  Functional Tests B2B

File: app/code/Magento/Analytics/Test/Mftf/ActionGroup/AdminConfigGeneralAnalyticsPageOpenActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Analytics/Test/Mftf/ActionGroup/AdminConfigGeneralAnalyticsPageOpenActionGroup.xml]: **Done**

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Review by rogyar — APPROVED** [review]

**Comment by rogyar** [comment]
Hi @sidolov. Could you approve this one, please? 
Thank you!

**Review by sidolov — APPROVED** [review]

**Comment by sidolov** [comment]
@magento run all tests

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-8061](https://jira.corp.magento.com/browse/ENGCOM-8061) has been created to process this Pull Request

**Pull request merged**